packages/graphql-testing/src/Requests.ts

```diff
@@ -0,0 +1,51 @@
+import {GraphQLRequest} from 'apollo-link';
+
+export default class Requests {
```

Comment:
Maybe this should just be called `Operations`, and then remove the word `Operation` from all the method names. It's a bit weird that we are using requests/ operations interchangeably.